<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RTDUTestClass</class>
 <widget class="QMainWindow" name="RTDUTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RTDUTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QDialogButtonBox" name="btnOKCancel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>450</y>
      <width>156</width>
      <height>23</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
    <property name="centerButtons">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>871</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Read</string>
    </property>
    <widget class="QPlainTextEdit" name="txtResult">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>70</y>
       <width>521</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblTagName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TagName(each line is a TagName):</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGetData">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Read Datapoint</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txtTagName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>311</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblResult">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Result(value + time of seconds + status):</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStartModbusServer">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>30</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Modbus Server</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>871</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Write</string>
    </property>
    <widget class="QLineEdit" name="leTagName">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>30</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblTagName_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TagName:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radType">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>开关量</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radType_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>模拟量</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leStatus">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblTagName_3">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Status(int):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leValue">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblTagName_4">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Value(double):</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnWriteData">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Write Datapoint</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RTDUTest.qrc"/>
 </resources>
 <connections/>
</ui>
